src/vmm/src/arch/aarch64/cache_info.rs

```diff
@@ -437,10 +425,7 @@ mod tests {
         let res = CacheEntry::from_index(0, engine.store.as_ref());
         assert!(res.is_err());
         // We did create the level file but we still do not have the type file.
-        assert_eq!(
-            format!("{}", res.unwrap_err()),
-            "Cannot proceed with reading cache info."
-        );
+        matches!(res.unwrap_err(), CacheInfoError::MissingCacheType);
```

Comment:
the original code would have panicked with the assert, is there any particular reason to change this behaviour?

Reply:
Yeah, I forgot to add `assert!` around `matches!`. Fixed now.